test/distributed/fsdp/test_fsdp_use_orig_params.py

```diff
@@ -961,53 +962,6 @@ def test_writeback_shape_mismatch(self):
 
 
 class TestFSDPUseOrigParamsFQNs(FSDPTest):
-    @skip_if_lt_x_gpu(2)
-    def test_param_and_buffer_names(self):
```

Comment:
Curious, why are we getting rid of this test as opposed to changing it to reflect the logical change here?

Reply:
IMHO, because we are not overriding `named_parameters()`, the unit test is not meaningful. It would be like writing a unit test for `CheckpointWrapper` or `FullyShardedDataParallel(use_orig_params=False)` that `named_parameters()` contains the prefix. This is directly testing the functionality of `nn.Module`.